sdk/core/Azure.Core.Experimental/src/MutableJsonDocument.cs

```diff
@@ -45,15 +45,14 @@ public MutableJsonElement RootElement
         /// <summary>
         /// Writes the document to the provided stream as a JSON value.
         /// </summary>
-        /// <param name="stream"></param>
-        /// <param name="format"></param>
-        /// <exception cref="ArgumentOutOfRangeException"></exception>
+        /// <param name="stream">The stream to write the document to.</param>
```

Comment:
@annelo-msft ?! Ending a sentence in a preposition. I'm shocked. Shocked!

Reply:
I knew you would call this when I did it! :)

Reply:
Fixed.